From 5b358d0b240fb44f2d6af5a6aafddc32149ff47b Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Sun, 19 May 2024 20:52:35 +1100
Subject: [PATCH] work around for warning that clang falsely treats as error

---
 .bazelrc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.bazelrc b/.bazelrc
index 92e73998..476b212d 100644
--- a/.bazelrc
+++ b/.bazelrc
@@ -410,7 +410,7 @@ common:ios --noenable_platform_specific_config
 common:android --copt=-w
 common:ios --copt=-w
 common:linux --host_copt=-w
-common:macos --copt=-w
+common:macos --copt=-w --copt=-Wno-c++11-narrowing
 common:windows --copt=/W0
 common:windows --host_copt=/W0
 
-- 
2.43.0

