From 008cd8e8d042ae7b93ae361c5a5ac8157387e296 Mon Sep 17 00:00:00 2001
From: David Lecomber <david+dslarm@lecomber.net>
Date: Fri, 30 Jan 2026 13:38:07 +0000
Subject: [PATCH] Force python wheel architecture to aarch64

---
 tensorflow/tools/pip_package/BUILD | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/tensorflow/tools/pip_package/BUILD b/tensorflow/tools/pip_package/BUILD
index 3437fa02..cd43d765 100644
--- a/tensorflow/tools/pip_package/BUILD
+++ b/tensorflow/tools/pip_package/BUILD
@@ -324,12 +324,7 @@ tf_wheel(
         "@platforms//os:windows": "win",
         "@platforms//os:linux": "linux",
     }),
-    platform_tag = select({
-        "@platforms//cpu:aarch64": "arm64",
-        "@platforms//cpu:arm64": "arm64",
-        "@platforms//cpu:x86_64": "x86_64",
-        "@platforms//cpu:ppc": "ppc64le",
-    }),
+    platform_tag = "arm64",
     source_files = [
         "MANIFEST.in",
         "//tensorflow/tools/pip_package:THIRD_PARTY_NOTICES.txt",
-- 
2.43.0

